### PR TITLE
Remove obsolete sentence

### PR DESCRIPTION
**Pull request by ligi**
We also need to care for:

> Once a maintainer has confirmed you have completed this step, they will merge your PR and send you a welcome email containing your invitations to the **CASA Call Calendar** and **Discord**.

the problem is we do not have the email at this point - and not sure it is good to add emails to this list - but it would actually be good to have the emails of members (e.g. to invite to calls / meetings)

### CODE REVIEW

**Comment by bumblefudge** [comment]
hmmmmmm, i've struggled with this.  we could set up an email list through whatever free service, but i'm not sure any of us wants the additional overhead of managing an email list any more than we want to manage a traditional website.  could we just instruct joiners people "Watch all activity" on the repo so that their github email functions as an email list? 
image

Or even recommend some custom settings?
image

**Review by bumblefudge — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -114,8 +114,6 @@ Below, please find a list of all members and their organizations.
 
 ## **Join CASA**
 
-CASA will begin meetings in 2021 as we wait for the initial working groups to form.
-
-1. Submit a Pull Request to this repository adding yourself as a member in the table above.
+Submit a Pull Request to this repository adding yourself as a member in the table above.
 
 Once a maintainer has confirmed you have completed this step, they will merge your PR and send you a welcome email containing your invitations to the **CASA Call Calendar** and **Discord**.
```
Comment (bumblefudge) [inline README.md:119]:
```suggestion
Once a maintainer has confirmed you have completed this step, they will merge your PR.  We recommend checking that your GH email is up-to-date and "Watching" all issues and pull requests in the /CASA/  repo, as well as /namespaces/ and/or /CAIPs/ to stay up-to-date.
```

Comment (ligi) [reply README.md:119]: The Gatherings are in the Gatherings repo - maybe we should ask to watch this one (also)?

File: README.md [hunk 2]
```diff
@@ -114,8 +114,6 @@ Below, please find a list of all members and their organizations.
 
 ## **Join CASA**
 
-CASA will begin meetings in 2021 as we wait for the initial working groups to form.
+Submit a Pull Request to this repository adding yourself as a member in the table above.
 
-1. Submit a Pull Request to this repository adding yourself as a member in the table above.
-
-Once a maintainer has confirmed you have completed this step, they will merge your PR and send you a welcome email containing your invitations to the **CASA Call Calendar** and **Discord**.
+Once a maintainer has confirmed you have completed this step, they will merge your PR.  We recommend checking that your GH email is up-to-date and "Watching" all issues and pull requests in the /CASA/  repo, as well as /namespaces/ and/or /CAIPs/ to stay up-to-date on editorial and in-person meetings.
```
Comment (ligi) [inline README.md:119]:
```suggestion
Once a maintainer has confirmed you have completed this step, they will merge your PR.  We recommend checking that your Github email is up-to-date and you are "Watching" these repos:
 *  [Gatherings](https://github.com/ChainAgnostic/Gatherings) to stay up to date with in person gatherings
 * [CASA](https://github.com/ChainAgnostic/CASA)  to stay up to date with the organisation and scheduled calls
  * [CAIPs](https://github.com/ChainAgnostic/CAIPs)
  * [namespaces](https://github.com/ChainAgnostic/namespaces)

```

**Pull request merged**